deployment/common/changeset/deploy_link_token.go

```diff
@@ -0,0 +1,26 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/changeset/internal"
+)
+
+var _ deployment.ChangeSet[uint64] = DeployLinkToken
+
+// DeployLinkToken deploys a link token contract to the chain identified by the chainSelector.
+func DeployLinkToken(e deployment.Environment, chainSelector uint64) (deployment.ChangesetOutput, error) {
```

Comment:
This is fine for now, but I think we want to move this over to the chainlink/deployment once we've stabilized it. Maybe.  Ideally stuff like this is reusable by other products' changesets when they're deploying to a chain for the first time. They can't reuse this in their deployment logic if it's here. 

But for this PR, and getting your sea legs, this is fine. We can move it over in a subsequent change. Have to find a good pattern for code organization for shared things there too, so better to do it separately.